### DISCUSSION TITLE
AutofacBootstrapper throwing error WHen PropertyChanged.Fody is active

### DISCUSSION THREAD

**Discussion by BigHoss** [canton7/Stylet#204] [Q&A]
Hey, 

as i hit thie problem with the second project and really don't want to write the whole ```SetAndNotify(() => asdf)``` fuzz again here is my question i can't seem to find an answer for.

I implemented the AutofacBootstrapper as specified in the wiki.
As soon as i install the fody package i get the following error on build:
```Autofac.Core.Activators.Reflection.NoConstructorsFoundException: No accessible constructors were found for the type 'BatchMoverUi_ProcessedByFody'.```

**Comment by canton7** [comment]
No idea I'm afraid -- I'm not sure why Fody is creating a type called `BatchMoverUi_ProcessedByFody`, and I don't know why Autofac might be trying to instantiate it.

**Comment by canton7** [comment]
Take a look at #211
